packages/framer-motion/src/animation/legacy-popmotion/spring.ts

```diff
@@ -109,35 +111,6 @@ export function spring({
                             initialDelta * Math.cos(angularFreq * t))
                 )
             }
```

Comment:
Here we're replacing the use of Motion One's `spring` function, which only supports under- and critically-dampened springs, with Framer Motion's which supports over-damped springs also.

To help offset the added bundlesize, we're replacing the closed-form velocity calculation with a second sample of the spring calculation.